nototools/generate_dingbats_html.py

```diff
@@ -667,6 +679,7 @@ def context_string(codelist, cp):
         line.append('<td>&nbsp;')
     line.append('<td class="age">%s' % unicode_data.age(cp))
     name = _flagged_name(cp, flag_sets)
+    name = _break_name(name)
     line.append('<td class="name">%s' % name)
```

Comment:
Thanks for the suggesstion.  Each table has a different number of columns, so I'd have to generate this data for each table separately, no?.  The first column and the last two columns can be dealt with this way, so I'll do that.